### ISSUE TITLE
Dogecoin Core version

### ISSUE THREAD

**Issue by langerhans** [trezor/blockbook#39]
Hey folks,

I was directed here by the customer support. I noticed that you now provide a full Dogecoin wallet interface along with the Trezor support that has been there before. As Dogecoin dev that made me wonder what version of Dogecoin Core you're running given ZMQ support is currently only available in the alpha version of 1.14. Maybe I'm misunderstanding but blockbook seems to rely on ZMQ connections to the coin daemon, no?

Anyway, if that is the case I'd be happy to have any feedback about that version given you'd be the first large scale service to run it.

**Comment by martinboehm** [comment]
Hi, you are very watchful :) 
We currently use the version 1.10.0 of Dogecoin Core and we are aware that ZMQ support is missing in this version.
We implemented in our Blockbook (independently of Dogecoin) a fallback mechanizm for the case that ZMQ fails, which polls the backend in periodical intervals. For the Dogecoin case we set up the polling intervals to 2 seconds for mempool and 30 seconds for new blocks, which replaces the ZMQ notifications quite well. When we upgrade to a version with ZMQ support, we will slow down the polling to our standard 1 minute for mempool and 15 minutes for new blocks.
